docs/pages/setup/guides/fluentd.mdx

```diff
@@ -44,11 +44,22 @@ from Teleport's events API, and forwards them to Fluentd.
   $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
   ```
   </TabItem>
+
   <TabItem label="Docker">
   ```code
   $ docker pull quay.io/gravitational/teleport-plugin-event-handler:(=teleport.plugin.version=)
   ```
   </TabItem>
+
+  <TabItem label="Helm">
+  ```code
+  $ helm repo add teleport https://charts.releases.teleport.dev/
+  $ helm install teleport-plugin-event-handler teleport/teleport-plugin-event-handler --values teleport-plugin-event-handler-values.yaml
```

Comment:
So the reader doesn't need to jump around the page, maybe we could limit this snippet to the `helm repo add` command, then explain that we will configure the plugin and run it later in the guide. We won't need to link to Step 6, since the reader will eventually get there.

I would recommend incorporating the second half of Step 6 (running the `teleport-event-handler start` command) into each of the "Teleport Cloud" and "Self-Hosted" `TabItem`s, then adding an example command to the "Helm Chart" `TabItem` in Step 6 to install the chart with the given values file.

Reply:
I moved it there now, but I feel like it's confusing if the other command (starting it directly) is immediately under the tab view. How about moving the command into the other tabs as well (Teleport Cloud and Self-hosted)?